selftests/functional/test_lv_utils.py

```diff
@@ -17,6 +18,8 @@ class LVUtilsTest(unittest.TestCase):
     Check the LVM related utilities
     """
 
+    @unittest.skipIf(sys.platform.startswith('darwin'),
+                     'macOS does not support LVM')
```

Comment:
Well is this necessary? I mean unless there is `vgs` command present it should state that vgs is not available.